Source/WebCore/page/Navigator.cpp

```diff
@@ -233,6 +234,24 @@ void Navigator::showShareData(ExceptionOr<ShareDataWithParsedURL&> readData, Ref
     });
 }
 
+// https://html.spec.whatwg.org/multipage/system-state.html#pdf-viewing-support
+// Section 8.9.1.6 states that if pdfViewerEnabled is true, we must return a list
+// of exactly five PDF view plugins, in a particular order.
+constexpr ASCIILiteral webKitBuiltInPDFPluginName { "WebKit built-in PDF"_s };
```

Comment:
It looks like this is used only in one place so we may not need a global variable for this.

Reply:
Good catch. I've removed it.